packages/webpack-config/package.json

```diff
@@ -66,5 +65,11 @@
     "style-loader": "1.0.1",
     "webpack": "4.41.2",
     "webpack-merge": "4.2.2"
+  },
+  "devDependencies": {
```

Comment:
i think all the jest-related stuff should be located in the root dir, to be later extended by the packages